good-citizen.md

```diff
@@ -0,0 +1,50 @@
+
+Words in themselves are meaningless. 
+They are vibrations caused by puffs of air coming out of our lungs.
+However, because of **context**, words are powerful.
+
+It's easy to know how you intend your words to come across, but it's difficult to know how others hear your words.
+What we say is strongly tempered by our experiences in life, our cultural norms, and our primary spoken language.
+At Exercism, we believe it's better to choose words that are **clear** and **unambiguous** across cultures and, where possible, across spoken languages.
+This means being careful when using metaphors, acronyms or phrases that might only be understandable to someone from a specific background or with specific knowledge.
+It also means being open to learning about how people from different cultures might misunderstand your intentions, and changing the words you use accordingly.
+
+One example is that within Exercism we ask people not to use the word "guys" unless explicitely addressing a group of men. 
+While in many places the word "guys" can refer to a group of people irrespective of gender, in many other places it explicitly means "men".
+So, regardless of the intention of the speaker, a person from that cultural background hearing the word "guys" understands the speaker to be only addressing men and, if they are not male, feels excluded.
+In tech, where men are in the strong majority, this adds yet another sense of negativity and exclusion to everyone else.
+
+At first, many people find this rule to be pedantic or irrational, as they are clear in their intentions to not exclude anyone.
+However, when guided by the principle of communicating in a way that is clear and unambiguous, and driven by the desire to be kind to others in the community, it becomes clear that the word "guys" does not necessarily convey the intended meaning to everyone, and so they choose to use clearer words.
+
+~~~~note
+Our co-founder, Jeremy, is from the UK, where the British dictionary explicitely states that "guys" is a gender neutral word.
+In real life, he commonly addresses groups of women using the word "guys". 
+However, in an international setting such as conferences, or within a multi-national platform such as Exercism, he would choose to use a clearer word such as "folks" (or occasionally y'all when he's feeling particularly Texan).
+As such, there is no judgement about whether the word "guys" should mean one thing or another or is good or bad, purely that the usage of the word should change with context.
+~~~~
+
+### You don't know what you don't know
+
+Just as it is important to say the right things, it is equally important to understand that people who cause offence might have had no intention to do so.
+Cultural differences mean that people often say things that would be clear to them and their friends to mean one thing, but mean something entirely different to you.
```

Comment:
Perhaps it's also worth mentioning language barriers. Some people may simply lack the vocabulary, eloquence or language skills to both be aware of the potentially hurtful meaning and, even if they are aware, to know how to phrase it differently to avoid it. For example, I've had ten years of English classes in school, yet the singular they wasn't mentioned a single time and it has no equivalent in German, so if I didn't learn about it "by accident" through picking it up from others, I'd not even be aware that it existed and would just use "he" by default.